routes/clients.rb

```diff
@@ -18,11 +18,33 @@ def self.registered(app)
       app.get '/clients/:id/?' do
         # view a client
         @client = Client[id: params[:id]]
+        @project = Project[id: params[:id]]
+        @node = NodeResource[id: params[:id]]
+        @db = DBResource[id: params[:id]]
```

Comment:
I don't think these are correct, you get the client's id in the params, but that isn't the same as the id of the project that belongs to that client. 

With reciprocal relationships, you should be able to just fetch the client, and then refer to its projects like @client.projects and the projects resources like project.node_resources..

see http://sequel.jeremyevans.net/rdoc/files/doc/association_basics_rdoc.html - especially Methods Added and Filtering by Associations sections